flipperdevices/Flipper-Android-App
# Add move-to into new file manager

**Background**

Move-to feature allows multiple or single files to be moved inside flipper zero file system. The feature itself contains limited functionality of Listing. Create folders, sort, show hidden files and etc what is often used inside move-to feature of other file managers.

**Changes**

- Add move-to feature for files
- Edit FlipperButton to add loader. Also made color change animated
- Increase timeout duration to 30 seconds

**Test plan**

- Open file manager, try move one file or multiple files
- Try the same for folders
- Try move parent folder inside parent's child folder and see it's impossible
